src/graphql/data/nft/NftBalance.ts

```diff
@@ -152,6 +169,7 @@ export function useNftBalanceQuery(
       },
       collection: asset?.collection,
       collectionIsVerified: asset?.collection?.isVerified,
+      collectionName: asset?.collection?.name,
```

Comment:
also a little redundant with asset_contract.name

Reply:
thank you T_T removed